### PR TITLE
8th Localisation Crusade 📜⚔

### PR DESCRIPTION
**Pull request by DukeJeans**
## Changelog
- Rewrote localisation of the Cult of the Damned's get-quest events.
- Rewrote localisation of the event where the Alliance changing its leader, german translation added.
- Completed translation of traits to german.
- Correction of a vanilla line in german.
- Translation of dragon-related localisation.
- Translation of councilor titles to german.

### CODE REVIEW

**Comment by zumbak04** [comment]
> @zumbak04 and @arithon, please add your current changes here.

Mmmkay, added mine.

**Comment by zumbak04** [comment]
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1026/commits/75f4cbebdba97d76d7353c557ba7070240236b44 merge was pretty hard, so NOTE: check it and [PR files](https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/1026/files) a few times.

File: localisation/00_DEwc_united_localisation.csv [hunk 1]
```diff
@@ -66,4 +66,5 @@ EVTDESCHFPB21098;Perhaps it is time I asked [From.GetTitledFirstName] to spend s
...
+generic_artifact_prefix_de_r;;;r;;;;;;;;;;;x
+generic_artifact_prefix_de_s;;;s;;;;;;;;;;;xEVTDESC_MNM_7057_NEW;The use of dark magic has corrupted me! The dark tendrils of [Root.GetDWDevil]'s magic has invaded my body, eroding mind and muscle alike. It is obvious that [Root.GetDWDevil] is keeping me alive, but at what cost?;L'utilisation de la magie noire m'a corrompu[Root.Get_E]�! Les vrilles mal�fiques de la magie [Root.GetDWDevilDuDela] ont envahi mon corps, �rodant � la fois mon esprit et mes muscles. Il est �vident que [Root.GetDWDevil] me maintient en vie, mais � quel prix�?;Der Gebrauch dunkler Magie hat mich verdorben! Die dunklen Ranken von [Root.GetDWDevil]s Magie sind in meinen K�rper eingedrungen und haben meinen Geist und meine Muskeln gleicherma�en beeinflusst. Es ist offensichtlich, dass [Root.GetDWDevil] mich am Leben h�lt, aber zu welchem Preis!?;;�Usar la magia oscura me ha corrompido! Los tent�culos oscuros de la magia de [Root.GetDWDevil] han invadido mi cuerpo, minando cuerpo y m�sculo por igual. Es obvio que [Root.GetDWDevil] me mantiene con vida, �pero a qu� precio?;;;;;;;;;x
```
Comment (MemeWizard) [inline localisation/00_DEwc_united_localisation.csv]: Something went wrong here?

Comment (arithon) [reply localisation/00_DEwc_united_localisation.csv]: seems like it, but I could easily correct this. done in next commit

Comment (MemeWizard) [reply localisation/00_DEwc_united_localisation.csv]: Just hope there is no more errors like this

File: localisation/00_zuwc_united_localisation.csv [hunk 2]
```diff
@@ -2665,85 +2662,85 @@ EVTDESC_WCWAR_708_murloc;They appeared from beneath the waves. In a matter of ho
...
...hey are already calling the Kor'Kron the "Warchief's Praetorian" or "personal guards", and there is a general consensus that it is unlikely that the Kor'kron will defend the common interests and beliefs of the Horde's other members.;;[target_warchief.GetBestName] hat den Beginn einer Anwerbungskampagne angek�ndigt, mit der die gr��ten Krieger der Horde f�r eine neue Elitegarde unter dem Kommando des Kriegsh�uptlings zusammengestellt werden sollen: die Kor'Kron. Diese Einheit aus mutigen und wilden Soldaten soll dem Schutz des eigentlichen Leitbildes der Horde und dessen, wof�r sie steht, dienen, allerdings mit einem kleinen Vorbehalt: Sie ist direkt dem Kriegsh�uptling unterstellt, [target_warchief.Getderdie] pers�nlich ihre Mitglieder fast ausschlie�lich aus [target_warchief.GetHerHis]en Mit-Orks ausw�hlt, was auch den anderen Mitgliedern der Horde nicht verborgen geblieben ist. Sie bezeichnen die Kor'Kron bereits als �Pr�torianer des Kriegsh�uptlings� oder �pers�nliche Wachen�, und es besteht ein allgemeiner Konsens dar�ber, dass es unwahrscheinlich ist, dass die Kor'kron die gemeinsamen Interessen und �berzeugungen der anderen Mitglieder der Horde verteidigen werden.;;;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_zuwc_united_localisation.csv]: @DukeJeans take a look at changes here. They look better than the current version?

File: localisation/00_zuwc_united_localisation.csv [hunk 3]
```diff
@@ -2869,82 +2866,82 @@ repentance_ask;The [Root.RelHead.GetTitle] will hear our plea for salvation.;Le
...
+GetLeaveDisbandCap_Disband;Disband;;Aufl�sen;;;;;;;;;;;x
+GetLeaveDisbandCap_Leave;Leave;;Verlassen;;;;;;;;;;;x
+GetLeaveDisband_disband;disband;;aufl�sen;;;;;;;;;;;x
+GetLeaveDisband_leave;leave;;verlassen;;;;;;;;;;;x
+found_horde_political_faction;Create the Horde Faction;;Erstellt die Horde;;;;;;;;;;;x
+found_alliance_political_faction;Create the Alliance Faction;;Erstellt die Allianz;;;;;;;;;;;x
+found_horde_political_faction_desc;You founded the Horde faction so that independent rulers can join your side as an ally or even as a vassal.;;Ihr habt die Horde gegr�ndet, damit unabh�ngige Herrscher als Verb�ndeter oder sogar als Vasallen auf eurer Seite stehen k�nnen.;;;;;;;;;;;x
+found_alliance_political_faction_desc;You founded the Alliance faction so that independent rulers can join your side as an ally or even as a vassal.;;Ihr habt die Allianz gegr�ndet, damit unabh�ngige Herrscher als Verb�ndeter oder sogar als Vasallen auf eurer Seite stehen k�nnen.;;;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_zuwc_united_localisation.csv]: Must be written in present simple.

Comment (arithon) [reply localisation/00_zuwc_united_localisation.csv]: done

**Pull request merged**